environs/config/config.go

```diff
@@ -177,6 +177,9 @@ const (
 	// FanConfig defines the configuration for FAN network running in the model.
 	FanConfig = "fan-config"
 
+	// BackupDirKey defines the key that will specify the temporal backup directory.
```

Comment:
... specify the backup working directory.

Reply:
Fixed